pkg/backupapi/recovery_point.go

```diff
@@ -44,6 +44,7 @@ type CreateRecoveryPointResponse struct {
 // CreateRecoveryPointRequest represents a request to create a recovery point.
 type CreateRecoveryPointRequest struct {
 	PolicyID string `json:"policy_id"`
+	Name     string `json:"name"`
```

Comment:
In the CreateRecoveryPointRequest has required recovery_point_type. The API docs for creating Recovery Point is here: https://ducpx.stoplight.io/docs/backupservice-api/API_Specs/reference/Public-API.v1.yaml/paths/~1agent~1backup-directories~1%7BbackupDirectoryId%7D~1recovery-points/post